batchlib/workflows/mean_and_sum_cell_analysis.py

```diff
@@ -34,8 +34,8 @@ def mean_and_sum_cell_analysis(config):
     merger(config.folder, config.folder, force_recompute=True)
 
     # 4.) run all the usual workflow summaries
-    identifiers = modify_identifiers(identifiers)
-    stat_names = [idf + stat_name.replace('serum_', '')
-                  for idf in identifiers for stat_name in DEFAULT_PLOT_NAMES]
-    name = "MeanAndSumCellAnalysisWorkflow"
-    workflow_summaries(name, config, t0, stat_names)
+    # identifiers = modify_identifiers(identifiers)
```

Comment:
@constantinpape I just need to figure out what to do with this. `workflow_summaries()` will be correctly invoked as part of `run_cell_analysis` for mean and sum, so why do we need to call it explicitly here as well?

Reply:
No for `mean_and_sum` `feature_identifers is not None`, so `workflow_summaries` will not be invoked.

Reply:
ok, I got it now. It's the merged table that also needs to be written to the DB. Ok I'll try to make it work on the DB job level

Reply:
exactly

Reply:
:white_check_mark: